rackcity/views/asset_views.py

```diff
@@ -194,17 +187,24 @@ def asset_add(request):  # need to make network and power connections here
             except MacAddressException as error:
                 failure_message += "Some mac addresses couldn't be saved. " + \
                     str(error)
-                return JsonResponse(
-                    {"failure_message": failure_message},
-                    status=HTTPStatus.BAD_REQUEST,
+            try:
+                save_power_connections(
```

Comment:
If mac addresses fail, keep track of the errors but still try to save power connections